Spore ModAPI/Spore/Simulator/cIdentityColorable.h

```diff
@@ -26,7 +26,7 @@
 
 namespace Simulator
 {
-	enum class IdentityColors
+	enum IdentityColors
```

Comment:
Is there a reason to remove `enum class`? Simple `enum`s do not have namespace protection, so now you would have `Simulator::Player`, etc. If it is necessary to remove enum class, then the names of the elements should change to be more unqique, for instance `kIdentityColorPlayer`.